build.gradle

```diff
@@ -165,6 +165,13 @@ configure(javaProjects) {
                 exclude 'com.google.code.findbugs:annotations'
             }
 
+            // Retrfit2
+            dependencySet(group: 'com.squareup.retrofit2', version: ext.versionOf('retrofit2')) {
+                entry 'retrofit'
+                entry 'adapter-guava'
```

Comment:
`adapter-guava` and `converter-jackson` are only used for unit tests. So users can use an adapter that they want to use.